docusaurus/docs/learn/quickstarts/network/help/quickstart-walkthrough.md

```diff
@@ -0,0 +1,314 @@
+cp "${ZITI_PKI}/${ZITI_HOSTNAME}-network-components/cas.pem" "${ZITI_PKI}/${ZITI_HOSTNAME}-edge/edge.cas.pem"
+```
+
+## Create Controller
+
+### Setup
+
+Declare some environment variables used to generate the controller config file. Some environment variables used were already set previously.
+
+- `ZITI_CTRL_ADVERTISED_ADDRESS` is the address for the controller’s control plane
+- `ZITI_CTRL_EDGE_ADVERTISED_ADDRESS` is the address for the HTTP API
+- `ZITI_CTRL_ADVERTISED_PORT` is the port for the control plane
+- `ZITI_CTRL_EDGE_ADVERTISED_PORT` is the port for the HTTP API
+
+The following are locations of PKI files.
+
+- `ZITI_PKI_CTRL_KEY`
+- `ZITI_PKI_CTRL_SERVER_CERT`
+- `ZITI_PKI_CTRL_CERT`
+- `ZITI_PKI_CTRL_CA`
+- `ZITI_PKI_EDGE_KEY`
+- `ZITI_PKI_EDGE_SERVER_CERT`
+- `ZITI_PKI_EDGE_CERT`
+- `ZITI_PKI_EDGE_CA`
+- `ZITI_PKI_SIGNER_KEY`
+- `ZITI_PKI_SIGNER_CERT`
+
+```
+export ZITI_CTRL_ADVERTISED_ADDRESS="${ZITI_HOSTNAME}"
+export ZITI_CTRL_EDGE_ADVERTISED_ADDRESS="${ZITI_HOSTNAME}"
+export ZITI_CTRL_ADVERTISED_PORT=8440
+export ZITI_CTRL_EDGE_ADVERTISED_PORT=8441
+
+export ZITI_PKI_CTRL_KEY="${ZITI_PKI}/${ZITI_CTRL_CA_NAME}/keys/${ZITI_NETWORK_COMPONENTS_PKI_NAME}.key"
+export ZITI_PKI_CTRL_SERVER_CERT="${ZITI_PKI}/${ZITI_CTRL_CA_NAME}/certs/${ZITI_NETWORK_COMPONENTS_PKI_NAME}-server.chain.pem"
+export ZITI_PKI_CTRL_CERT="${ZITI_PKI}/${ZITI_CTRL_CA_NAME}/certs/${ZITI_NETWORK_COMPONENTS_PKI_NAME}-client.cert"
+export ZITI_PKI_CTRL_CA="${ZITI_PKI}/${ZITI_CTRL_CA_NAME}/cas.pem"
+
+export ZITI_PKI_EDGE_KEY="${ZITI_PKI}/${ZITI_EDGE_CA_NAME}/keys/${ZITI_EDGE_API_PKI_NAME}.key"
+export ZITI_PKI_EDGE_SERVER_CERT="${ZITI_PKI}/${ZITI_EDGE_CA_NAME}/certs/${ZITI_EDGE_API_PKI_NAME}-server.chain.pem"
+export ZITI_PKI_EDGE_CERT="${ZITI_PKI}/${ZITI_EDGE_CA_NAME}/certs/${ZITI_EDGE_API_PKI_NAME}-client.cert"
+export ZITI_PKI_EDGE_CA="${ZITI_PKI}/${ZITI_EDGE_CA_NAME}/edge.cas.pem"
+
+export ZITI_PKI_SIGNER_KEY="${ZITI_PKI}/${ZITI_SIGN_CA_NAME}/keys/${ZITI_SIGN_CA_NAME}.key"
+export ZITI_PKI_SIGNER_CERT="${ZITI_PKI}/${ZITI_SIGN_CA_NAME}/certs/${ZITI_SIGN_CA_NAME}.chain.pem"
+```
+
+### Create the Controller Config File
+
+The controller config file is populated based on the values of environment variables set up to this point.
+
+```
+"${ZITI_BIN_DIR}/ziti" create config controller >${ZITI_HOME}/${ZITI_HOSTNAME}.yaml
+```
+
+### Initialize the Controller
+
+Initializing the controller sets up database files and some of the configuration values.
+
+```
+mkdir ${ZITI_HOME}/db
+"${ZITI_BIN_DIR}/ziti" controller edge init "${ZITI_HOME}/${ZITI_HOSTNAME}.yaml" -u "admin" -p $ZITI_PWD
+```
+
+### Run the Controller
+
+```
+"${ZITI_BIN_DIR}/ziti" controller run ${ZITI_HOME}/${ZITI_HOSTNAME}.yaml &> ${ZITI_HOME}/${ZITI_HOSTNAME}.log &
+```
+
+### Wait for the Controller
+
+We want to ensure the controller is ready before creating a router, this mini loop
```

Comment:
Definitely don't use 'we' or "we'll" in the doc. You don't need to explain the loop, just that it's important for the controller to be online before proceeding

Reply:
you should explain WHY the controller needs to be online though.